### PR TITLE
Contact Section

### PR DESCRIPTION
**Pull request by SabiMantock**
Contact Section main features:

- Added contact section to index.html page
- Added styling to the contact section
- Added images to the portfolio

image

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -301,5 +301,53 @@ <h3 class="langTitle">Skills</h3>
...
+          id="form_container"
+          method="post"
+          action="https://formspree.io/f/xknepely"
+        >
+          <div class="input_form">
+            <input
+              type="text"
+              name="name"
+              placeholder="Name"
+              value="Yeremias"
```
Comment (Qoosim) [inline index.html:323]: - [ ] Kindly remove the value from your inputs. Doing so will make your form to submit once click the submit button without the input values from the potential users.

File: index.html [hunk 2]
```diff
@@ -301,5 +301,53 @@ <h3 class="langTitle">Skills</h3>
...
+              maxlength="30"
+              required
+            />
+          </div>
+          <div class="input_form">
+            <input
+              type="email"
+              name="email"
+              placeholder="Email"
+              value="natajayanj14@gmail.com"
```
Comment (Qoosim) [inline index.html:333]: - [ ] Kindly remove the value from your inputs. Doing so will make your form to submit once click the submit button without the input values from the potential users.

File: index.html [hunk 3]
```diff
@@ -301,5 +301,53 @@ <h3 class="langTitle">Skills</h3>
...
+              required
+            />
+          </div>
+          <div class="input_form">
+            <textarea
+              name="message"
+              placeholder="Write your message here"
+              maxlength="500"
+            ></textarea>
```
Comment (Qoosim) [inline index.html:342]: - [ ] Every inputs must have a required attribute. Kindly add required attribute to your `textarea` input.

**Review by Qoosim — CHANGES_REQUESTED** [review]
#  *Status: Required Changes* :recycle:

Hi @SabiMantock, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

-  No linter errors. :heavy_check_mark: 
-  Good README file. :heavy_check_mark: 
-   Correct GitHub Flow. :heavy_check_mark: 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (SabiMantock) [reply index.html]: thank you for the feedback, will make the appropriate changes.

File: index.html (hunk 3 above)
Comment (SabiMantock) [reply index.html]: thank you for the feedback, will make the appropriate changes.

File: index.html (hunk 2 above)
Comment (SabiMantock) [reply index.html]: thank you for the feedback, will make the appropriate changes.

**Review by PinkMoon25 — APPROVED** [review]
Hi @SabiMantock , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
No linter errors. :heavy_check_mark:
Good README file. :heavy_check_mark:
Correct GitHub Flow. :heavy_check_mark:

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**